### DISCUSSION TITLE
Updated version differs from expected one

### DISCUSSION THREAD

**Discussion by rombert** [renovatebot/renovate#22268] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate should update the Tika version to 2.7.0, which is the latest available version.

https://github.com/apache/sling-org-apache-sling-starter/pull/148 claims to update the tika dependencies from 1.28.5 to 2.7.0.  However, the diff shows that the version is set to 2.8.0

```diff
diff --git a/src/main/features/base.json b/src/main/features/base.json
index 66ec3d2..21780d8 100644
--- a/src/main/features/base.json
+++ b/src/main/features/base.json
@@ -211,11 +211,11 @@
             "start-order":"10"
         },
         {
-            "id":"org.apache.tika:tika-core:1.28.5",
+            "id":"org.apache.tika:tika-core:2.8.0",
             "start-order":"10"
         },
         {
-            "id":"org.apache.tika:tika-parsers:1.28.5",
+            "id":"org.apache.tika:tika-parsers:2.8.0",
             "start-order":"10"
         },
         {
```

The tika-parsers version present on Maven Central at https://repo.maven.apache.org/maven2/org/apache/tika/tika-parsers/2.8.0/ has only a POM file, so it can't be used for updates.

### What you tried so far.

Nothing, I have no idea how to influence the version chosen by Renovate.

### Relevant debug logs

<details><summary>Logs</summary>
```
{
            "datasource": "maven",
            "depName": "org.apache.tika:tika-core",
            "currentValue": "1.28.5",
            "updates": [
              {
                "bucket": "major",
                "newVersion": "2.8.0",
                "newValue": "2.8.0",
                "releaseTimestamp": "2023-05-11T19:28:31.000Z",
                "newMajor": 2,
                "newMinor": 8,
                "updateType": "major",
                "branchName": "renovate/major-apache-tika"
              }
            ],
            "packageName": "org.apache.tika:tika-core",
            "warnings": [],
            "versioning": "maven",
            "sourceUrl": "https://github.com/apache/tika",
            "registryUrl": "https://repo.maven.apache.org/maven2",
            "homepage": "https://tika.apache.org/",
            "currentVersion": "1.28.5",
            "isSingleVersion": true,
            "fixedVersion": "1.28.5"
          },
          {
            "datasource": "maven",
            "depName": "org.apache.tika:tika-parsers",
            "currentValue": "1.28.5",
            "updates": [
              {
                "bucket": "major",
                "newVersion": "2.8.0",
                "newValue": "2.8.0",
                "releaseTimestamp": "2023-05-11T19:28:55.000Z",
                "newMajor": 2,
                "newMinor": 8,
                "updateType": "major",
                "branchName": "renovate/major-apache-tika"
              }
            ],
            "packageName": "org.apache.tika:tika-parsers",
            "warnings": [],
            "versioning": "maven",
            "sourceUrl": "https://github.com/apache/tika",
            "registryUrl": "https://repo.maven.apache.org/maven2",
            "homepage": "https://tika.apache.org/",
            "currentVersion": "1.28.5",
            "isSingleVersion": true,
            "fixedVersion": "1.28.5"
          },
```
</details>

**Comment by rarkins** [comment]
Why does that 2.8.0 exist at all - is it a publishing mistake? It's also "latest" in https://repo.maven.apache.org/maven2/org/apache/tika/tika-parsers/maven-metadata.xml

Would this mean that Renovate needs to check for the existence of an e.g. JAR file?

**Reply by rombert** [reply]
It seems that there was a change regarding that module between 1.x and 2.x. Compare https://repo.maven.apache.org/maven2/org/apache/tika/tika-parsers/1.28.5/ with https://repo.maven.apache.org/maven2/org/apache/tika/tika-parsers/2.0.0 .

I don't think Renovate should change how it checks releases. What threw me off was that the PR and the commit mention updating to 2.7.0, which is inconsistent with the code change. It would be very good if the PR, the commit, and the code change are aligned.

**Comment by rarkins** [comment, marked as answer]
I created #22270 as a bug report about this. Clicking rebase/retry should result in the correct PR body being updated now.

**Reply by rombert** [reply]
Thanks!
